core/controllers/classifier_test.py

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 """Tests for the controllers that communicate with VM for training
-classifiers."""
+classifiers.
+"""
```

Comment:
Why have the triple quotes been moved to the next line? Is it part of the test and is the expected behavior?

Reply:
This is the standard (according to pep8): https://www.python.org/dev/peps/pep-0008/#documentation-strings